packages/eas-cli/src/commandUtils/EasCommand.ts

```diff
@@ -45,6 +47,20 @@ export type ContextOutput<
 
```

Comment:
Add utility type helper for inline comment below:

```suggestion

type GetContextType<Type> = {
  [Property in keyof Type]: any;
};

```